templates/registration/profile.html

```diff
@@ -7,15 +7,15 @@
 
 {% block content %}
     <h1>Profile</h1>
-    <h2 class="profile-header">Good morning {% firstof user.get_full_name user.get_username %}!</h2>
```

Comment:
Actually nvm, I just noticed this branch was built on top of that one. This is fine.